### DISCUSSION TITLE
Ray remote connect to subsites directory

### DISCUSSION THREAD

**Discussion by vdvcoder** [spatie/ray#524] [Q&A]
Hello,

Is it possible to remote connect to a server but to a subsites directory?
I have a project that is on a shared hosting, and the staging is on a subdomain that is in the following folder 'subsites/projectname'

I assume that ray remote now only connects to the project in the www folder?

Thanks in advance!

**Comment by AdrianMrn** [comment]
Hi @vdvcoder, the folder structure shouldn't matter, you just need SSH access to the server. Let me know if you need any more help!
